src/index.js

```diff
@@ -1 +1,92 @@
+/* eslint-disable no-console */
 'use strict';
+
+const http = require('http');
+const url = require('url');
+const fs = require('fs');
+const formidable = require('formidable');
+const path = require('path');
+const { getCompressionType } = require('./modules/compressionType');
+const { pipeline } = require('stream');
+
+const PORT = process.env.PORT || 3000;
+const server = new http.Server();
+
+server.on('request', (req, res) => {
+  const normalizedURL = new url.URL(req.url, `http://${req.headers.host}`);
+  const pathName = normalizedURL.pathname.slice(1) || 'index.html';
+  const filePath = path.resolve('public', pathName);
+
+  if (pathName === 'index.html') {
+    if (!fs.existsSync(filePath)) {
+      res.statusCode = 404;
+      res.end('Page not found');
```

Comment:
can be less
```suggestion
      res.status(404).end('Page not found');
```

Reply:
It's doesn't work. Maybe this will be working in express?